src/tenant/fixtures/tenants.json

```diff
@@ -3,7 +3,7 @@
     "model": "tenant.tenant",
     "pk": 1,
     "fields": {
-      "domain_url": "localhost",
+      "domain_url": "bytedeck.com",
```

Comment:
leave as localhost or even better, 127.0.0.1, because that's easier for developers, and also where django tells you to go when you runserver